lib/libraries/c3/extensions/default-date-format.js

```diff
@@ -0,0 +1,32 @@
+module.exports = function(startDate, endDate){
+  var timeDiff = Math.abs(new Date(startDate).getTime() - new Date(endDate).getTime());
+  var months = [
+    'Jan', 'Feb', 'Mar',
+    'Apr', 'May', 'June',
+    'July', 'Aug', 'Sept',
+    'Oct', 'Nov', 'Dec'
```

Comment:
Any specific benefit to doing 3 per line here? 12 lines is long, but it's easy to scan and consistent with how long arrays are generally handled elsewhere in all our code (I think?).